app/src/main/java/com/framgia/englishconversation/utils/binding/BindingUtils.java

```diff
@@ -12,6 +12,7 @@
 import android.text.Editable;
 import android.text.TextWatcher;
 import android.text.format.DateUtils;
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports